src/bikeracks/votes.py

```diff
@@ -68,7 +68,7 @@ def submit_vote():
     
     # connect to db
     db = get_db()
-    
+                
```

Comment:
There's generally no reason to have trailing whitespace in your source code.

 A linter (e.g. pylint) would probably complain about this.